roles/confluent.kerberos/templates/krb5.conf.j2

```diff
@@ -1,14 +1,8 @@
-[logging]
- default = FILE:/var/log/krb5libs.log
- kdc = FILE:/var/log/krb5kdc.log
- admin_server = FILE:/var/log/kadmind.log
-
 [libdefaults]
  default_realm = {{ kerberos.realm|upper() }}
  dns_lookup_realm = {{ kerberos.dns_lookup_realm }}
  dns_lookup_kdc = {{ kerberos.dns_lookup_kdc }}
  ticket_lifetime = {{ kerberos.ticket_lifetime }}
- renew_lifetime = {{ kerberos.renew_lifetime }}
```

Comment:
@JumaX any idea why this prop was messing everything up for me?

Reply:
I suppose I should remove the variable if i am removing it from the template

Reply:
We probably shouldn't remove the renew lifetime, it's actually pretty important.  Ticket lifetime refers to the life of the given ticket, as in its valid for 10 hours and you can use it to auth during that ten hour period.  The renewal lifetime states that if the ticket has expired, but hasn't exceeded the renewal lifetime, it can be used to request a new ticket from the KDC.  So it's pretty important to the kerberos life cycle.  Not sure why it would have caused you problems, I suspect that maybe something changed in the version of the KDC you were/are running?  I had a similar issue with OpenLDAP, where I had to roll back to an older version to get something to work, as I couldn't be bothered to figure out what else had changed in the new release.  I think their is a lesson for both of us here, where we should specific specific version of all packages to be installed, including for ancillary components like KDC and LDAP.

Reply:
According to this: https://stackoverflow.com/questions/38555244/how-do-you-set-the-kerberos-ticket-lifetime-from-java
renewable_lifetime and ticket_lifetime are not honored in the version of java we are using w cp ansible

Reply:
@domenicbove Amazing catch!  We also aren't supporting Java 9, I think the next version we are support is 12.  It's confusing, but Java's release schedule has changed to be every 6 months now, thus we've skipped over Java 9.  Let's remove the property then.